### PR TITLE
Update to use version 2.4.5 of the weld osgi bundle

### PR DESCRIPTION
**Pull request by benjamin-confino**

### CODE REVIEW

File: dev/com.ibm.ws.org.jboss.weld/bnd.bnd [hunk 1]
```diff
@@ -8,8 +8,8 @@
 # Contributors:
 #     IBM Corporation - initial API and implementation
 #*******************************************************************************
--include= jar:${fileuri;${repo;org.jboss.weld:weld-osgi-bundle;[2.4.3.Final,3)}}!/META-INF/MANIFEST.MF,bnd.overrides
+-include= jar:${fileuri;${repo;org.jboss.weld:weld-osgi-bundle;[2.4.5.Final,5)}}!/META-INF/MANIFEST.MF,bnd.overrides
```
Comment (tevans78) [inline dev/com.ibm.ws.org.jboss.weld/bnd.bnd]: should be `[2.4.5.Final,3)`

File: dev/com.ibm.ws.org.jboss.weld/bnd.overrides [hunk 2]
```diff
@@ -72,6 +72,6 @@ Service-Component: \
         |org.jboss.weld.util.bean"
 
 Include-Resource: \
-  @${repo;org.jboss.weld:weld-osgi-bundle;[2.4.3.Final,3)}!/!META-INF/maven/*
+  @${repo;org.jboss.weld:weld-osgi-bundle;[2.4.5.Final,5)}!/!META-INF/maven/*
```
Comment (tevans78) [inline dev/com.ibm.ws.org.jboss.weld/bnd.overrides]: should be `[2.4.5.Final,3)`

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#build

**Pull request merged**